lib/pages/dashboard/search/search_screen.dart

```diff
@@ -8,84 +8,95 @@ class SearchScreen extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      backgroundColor: MyColors.blackColor,
-      resizeToAvoidBottomInset: false,
-      body: SafeArea(
-        child: Padding(
-          padding: const EdgeInsets.symmetric(horizontal: 20),
-          child: CustomScrollView(
-            slivers: [
-              const _SearchBox(),
-              const SliverToBoxAdapter(
-                child: Padding(
-                  padding: EdgeInsets.only(top: 15, bottom: 20),
-                  child: Text(
-                    "Recent searches",
-                    style: TextStyle(
-                      fontFamily: "AM",
-                      fontWeight: FontWeight.w400,
-                      color: MyColors.whiteColor,
-                      fontSize: 17,
+    return GestureDetector(
+      onTap: () {
+        FocusScope.of(context).unfocus(); // Dismiss keyboard on tap outside
+      },
```

Comment:
Remove search bar to simplify things. To make it look natural copy UI from spotify from playlist page